_tutorials/new-component/create-a-new-component-part-2.md

```diff
@@ -0,0 +1,268 @@
+
+Fortunately, `o-buttons` allows us to output a button with a custom class name using the Sass mixin [`oButtonsContent`](https://registry.origami.ft.com/components/o-buttons@6.0.14/sassdoc?brand=master#mixin-obuttonscontent). There are a number of options we could pass to `oButtonsContent` for different types of buttons. We'll choose a "primary" button type for this tutorial. In `main.scss` define a new CSS class `.o-example__button` and include the button CSS with `oButtonsContent`:
+
+<pre><code class="o-syntax-highlight--scss">.o-example {
+	// [previously discussed css here]
+}
+
+.o-example__button {
+	@include oButtonsContent($opts: ('type': 'primary'));
+}</code></pre>
+
+We need to update our demo markup  `demos/src/demo.mustache` with the new button markup, similar to how we added text to the demo in [part one](/docs/tutorials/create-a-new-component-part-1):
+
+<pre><code class="o-syntax-highlight--diff">&lt;div class="o-example" data-o-component="o-example">
+	Hello world, I am a component named o-example!
++	&lt;button class="o-example__button">count&lt;/button>
+&lt;/div></code></pre>
+
+_Note: the double underscore in `.o-example__button` is part of the [BEM naming convention](https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/) mentioned earlier._
+
+<figure>
+	<img alt="" src="/assets/images/tutorial-new-component/hello-world-demo-4-basic-styles.png" />
+	<figcaption class="o-typography-caption">
+        Our "o-example" component now has a button.
+	</figcaption>
+</figure>
+
+## Part Three: Themes And Branding
+
+To style our components we covered many topics in this part of the tutorial. We learnt:
+- Origami component CSS is written with [Sass](https://sass-lang.com/documentation).
+- Component Sass includes [SassDoc](http://sassdoc.com/) comments for Sass documentation.
+- Conventional Origami Sass patterns such as the ["primary mixin"](/spec/v1/sass/#primary-mixin) and ["silent mode"](/spec/v1/sass/#sass-silent-mode).
+- How to install Origami component dependencies from the [Origami Bower Registry](https://github.com/Financial-Times/origami-bower-registry).
+- And finally how to include and use Sass from `o-colors`, `o-spacing`, `o-typography`, and `o-buttons`.
+
+Now we know how to add styles in part three we will build on that knowledge to provide new visual variations of our component. We will add an alternative `inverse` theme that will optionally modify the appearance of `o-example` to look better on a dark background. We will also "brand" our component to change its appearance depending on whether it is used within a reader facing ft.com project, internal project, or elsewhere. [Continue to part three](/docs/tutorials/create-a-new-component-part-3).
```

Comment:
```suggestion
Now we know how to add styles, in part three we will build on that knowledge to provide new visual variations of our component. We will add an alternative `inverse` theme that will modify the appearance of `o-example` to look better on a dark background. We will also "brand" our component to change its appearance depending on whether it is used within a reader facing ft.com project, internal project, or elsewhere. [Continue to part three](/docs/tutorials/create-a-new-component-part-3).
```

Reply:
The "Continue to part X." links should perhaps be a pagination layout of buttons --> https://registry.origami.ft.com/components/o-buttons@6.0.15#demo-pagination-layout

Reply:
I'm not sure pagination is usually used for a list of items rather than a continuation of a single article. 

But also I don't fancy setting this up in jekyll 🙈

Reply:
Do you think a comma is needed here? That's not how I'd say that sentence. I feel the pause makes it read a little awkwardly.